### DISCUSSION TITLE
Habitat.config error report

### DISCUSSION THREAD

**Discussion by JaechangKo** [facebookresearch/habitat-lab#1066] [Q&A]
For this tutorial session: https://www.youtube.com/watch?v=KbbfD7RlJLM&list=PLGywud_-HlCORC0c4uj97oppQrGiB6JNy
When I run the code 'Path Setup and Imports', habitat.config error doesn't allow me to move forward. Is this happened because of the incompatibility of TensorFlow version?  

Colab Link: [http://bit.ly/habitat_lab_interactive_tasks](url)
ImportError: cannot import name 'Config' from 'habitat.config' (/content/habitat-lab/habitat-lab/habitat/config/__init__.py)

---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
[<ipython-input-8-3997d7e4a0e6>](https://localhost:8080/#) in <module>
     26 import habitat
     27 import habitat_sim
---> 28 from habitat.config import Config
     29 from habitat.core.registry import registry
     30 from habitat_sim.utils import common as ut

ImportError: cannot import name 'Config' from 'habitat.config' (/content/habitat-lab/habitat-lab/habitat/config/__init__.py)

---------------------------------------------------------------------------
NOTE: If your import is failing due to a missing package, you can
manually install dependencies using either !pip or !apt.

To view examples of installing some common dependencies, click the
"Open Examples" button below.
---------------------------------------------------------------------------

![11](https://user-images.githubusercontent.com/10793897/211241815-67e2216c-52f7-4e6d-bc38-eee0304389e8.JPG)

![123](https://user-images.githubusercontent.com/10793897/211985164-4e0dfbec-4507-4ff3-8315-11ad206f356d.JPG)

**Comment by aclegg3** [comment]
Hey @JaechangKo 

I think Colabs are broken at the moment due to changes on their backend. We'll need to do some refactoring to fix that eventually. In the meantime, I suggest you install from conda or source and run the python locally. :slightly_smiling_face:
